src/olivePage.ts

```diff
@@ -44,6 +44,11 @@ export default class OlivePage {
         SystemExtensions.initialize();
         Modal.initialize();
 
+        //ASP.NET needs this config for Request.IsAjaxRequest()
+        $.ajaxSetup({
+            headers: { 'X-Requested-With': 'XMLHttpRequest' }
```

Comment:
If we didn't add this configuration, `Request.IsAjaxRequest()` is always **False**. this is especially needed in our Root Template.